CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## master / unreleased
 
+* [FEATURE] Implement an ability to load multiple Cortex modules using a comma-separated list. #3272
+  - Should be backwards compatible with the current behavior.
+  - Target list like 'all,compactor' can be used to load the single-binary modules plus some other modules.
```

Comment:
I suggest a little rephrasing to simplify it:

```suggestion
* [ENHANCEMENT] Allow to specify multiple comma-separated Cortex services to `-target` CLI option (or its respective YAML config option). For example, `-target=all,compactor` can be used to start Cortex single-binary with compactor as well. #3272
```

We consider these improvements "enhancement" and not new core features. Please move the changelog entry to the right place (entries are grouped by type).

Reply:
Sure, forgot that they're grouped.